src/Compilers/VisualBasic/Portable/PublicAPI.Unshipped.txt

```diff
@@ -0,0 +1,6 @@
+Microsoft.CodeAnalysis.VisualBasic.Syntax.SyntaxNodeRemover.New() -> Void
```

Comment:
I'd prefer the API review team take a look at these (possibly prior to us shipping 1.3), and decide if these are reasonable to be public or need a private constructor. They should automatically move over to shipped.txt with 1.3 release.